covid19-etl/etl/cityofchicago.py

```diff
@@ -0,0 +1,410 @@
+            "other_non_latinx": "Other race",
+            "other_race_non_latinx": "Other race",
+            "unknown_race_eth": "Unknown",
+            "unknown_race_ethnicity": "Unknown",
+        }
+
+        # race mapping for value from original dataset to value in Gen3 data dictionary
+        race = {
+            "latinx": "Hispanic",
+            "asian_non_latinx": "Asian",
+            "black_non_latinx": "Black",
+            "white_non_latinx": "White",
+            "other_non_latinx": "Other",
+            "other_race_non_latinx": "Other",
+            "unknown_race_eth": "Unknown",
+            "unknown_race_ethnicity": "Unknown",
+        }
+
+        # Gender for value from original dataset to value in Gen3 data dictionary
+        gender = {
+            "female": "Female",
+            "male": "Male",
+            "unknown_gender": "Unknown",
+        }
+
+        record_type_mapping = {
+            "cases": "count",
+            "deaths": "deaths",
+            "hospitalizations": "hospitalizations",
+        }
+
+        record_type, submitter_value = header_value.split("_", maxsplit=1)
+        if submitter_value in age_group:
+            if "80" in submitter_value:
+                age_group_dict["AgeGroup"] = "80+"
+            else:
+                age_group_dict["AgeGroup"] = age_group[submitter_value].replace(
+                    " to ", "_"
+                )
+            summary_group_demographics_value_dict["age_group"] = age_group[
+                submitter_value
+            ]
+
+        elif submitter_value in race:
+            age_group_dict["Race"] = race_submitter_id[submitter_value]
+            summary_group_demographics_value_dict["race"] = race[submitter_value]
+            if submitter_value == "latinx":
+                summary_group_demographics_value_dict["ethnicity"] = "Hispanic"
+            else:
+                summary_group_demographics_value_dict["ethnicity"] = "Nonhispanic"
+
+        elif submitter_value in gender:
+            age_group_dict["Gender"] = gender[submitter_value]
+            if submitter_value == "unknown_gender":
+                summary_group_demographics_value_dict[
+                    "gender"
+                ] = "Unknown or Left Blank"
+            else:
+                summary_group_demographics_value_dict["gender"] = gender[
+                    submitter_value
+                ]
+
+        else:
+            return
+
+        summary_group_demographics_submitter_id = derived_submitter_id(
+            summary_clinical_submitter_id,
+            "summary_clinical",
+            "summary_group_demographic",
+            age_group_dict,
+        )
+
+        if (
+            summary_group_demographics_submitter_id
+            not in self.summary_group_demographics
+        ):
+            self.summary_group_demographics[
+                summary_group_demographics_submitter_id
+            ] = summary_group_demographics_value_dict
```

Comment:
i don't understand this 🤔 you're calling `add_to_summary_group_demographics` for each column, and only storing `summary_group_demographics_value_dict` the 1st time. Doesn't that mean you're missing some values?

Reply:
oh i see, i think i missed [this](https://github.com/uc-cdis/covid19-tools/blob/5d113e05da7f37afab4cba977286f13098102e6b/covid19-etl/etl/cityofchicago.py#L234-L236) last time

Reply:
Ya no worries :)